src/models/point/model_point.py

```diff
@@ -123,6 +123,30 @@ def validate_history_interval(self, _, value):
             raise ValueError("history_interval needs to be at least 1, default is 15 (in minutes)")
         return value
 
+    @validates('input_min')
+    def validate_input_min(self, _, value):
+        if value is not None and self.input_max is not None and value > self.input_max:
+            raise ValueError("input_min cannot be greater than input_max")
+        return value
```

Comment:
`validate` function calls in sequential order, first to last. And if it's validated successfully, it will assign value on the object `self`. So in this function, `self.input_max` never be assigned and `self.input_min` is about to assign but it's input value will be on `value`.

So, for the validation of `input_min` & `input_max`, choose `@validate('input_max')`, where you have, `self.input_min` and `value` as `self.input_max`.